files/default/neutron-ha-tool.py

```diff
@@ -81,6 +81,9 @@ def parse_args():
                     help='Replicate DHCP configuration to all agents')
     ap.add_argument('--now', action='store_true', default=False,
                     help='Migrate Routers immediately without a delay.')
+    ap.add_argument('--wait-for-ports', action='store_true', default=False,
```

Comment:
@aspiers Ok. I changed the default. With that I believe I addressed all the concerns you raised.